Makefile

```diff
@@ -2,39 +2,45 @@
 # with Go source code. If you know what GOPATH is then you probably
 # don't need to bother with make.
 
-.PHONY: geth all test lint clean devtools help
+.PHONY: geth all test lint fmt clean devtools help
 
 GOBIN = ./build/bin
 GO ?= latest
 GORUN = go run
+GOFMT ?= gofmt "-s"
+GOFILES := $(shell find . -name "*.go")
```

Comment:
It's not necessary to define these as variables.

Reply:
Fixed in [c8e205a](https://github.com/ethereum/go-ethereum/pull/29777/commits/c8e205a5e46f51e1e66bd09e9308fc1fb3cbb731).